<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagSelectorWidget</class>
 <widget class="QWidget" name="TagSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbKey"/>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cbValue"/>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="btOr">
       <property name="text">
        <string>OR</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="btAnd">
       <property name="text">
        <string>AND</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="btNot">
       <property name="text">
        <string>NOT</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="btDel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/AllIcons.qrc">
         <normaloff>:/Icons/actions/edit_delete.png</normaloff>:/Icons/actions/edit_delete.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>is</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>isoneof</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons/AllIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btDel</sender>
   <signal>clicked()</signal>
   <receiver>TagSelectorWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>removed()</signal>
  <signal>and()</signal>
  <signal>or()</signal>
  <signal>not()</signal>
 </slots>
</ui>
